src/js/tests/api-client/apiClient.test.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-undef  */
```

Comment:
this rule will be no longer required after #278 get merged

Reply:
@kkaczmarczyk i know, but its not merged yet